search-app/src/cmr/search/services/query_execution/facets/facets_v2_results_feature.clj

```diff
@@ -44,14 +46,19 @@
 (def v2-facets-result-field-in-order
   "Defines the v2 facets result field in order"
   ["Keywords" "Platforms" "Instruments" "Organizations" "Projects" "Processing levels"
-   "Output File Formats" "Reprojections"])
+   "Measurements" "Output File Formats" "Reprojections"])
 
 (defn- facet-query
   "Returns the facet query for the given facet field"
   [facet-field size query-params]
-  (if (= :science-keywords facet-field)
+  (case  facet-field
+    :science-keywords
```

Comment:
I was curious so I looked it up. It looks like you could can group together the expressions if they have the same result.

```
(case facet-field
  (:science-keywords :variables)
  (let [depth (hv2/get-depth-for-hierarchical-field query-params query-params facet-field)]
    (hv2/nested-facet (facets-v2-params->elastic-fields facet-field) size depth)
  ;; else
```